oioswift/common/wsgi.py

```diff
@@ -25,6 +25,8 @@ def oio_make_env(env, *args, **kwargs):
     newenv = orig_make_env(env, *args, **kwargs)
     newenv['oio.query'] = env.get('oio.query', {}).copy()
     newenv['oio.cache'] = env.get('oio.cache')
+    newenv['oio.ephemeral_object'] = env.get('oio.ephemeral_object')
+    newenv['oio.list_mpu'] = env.get('oio.list_mpu')
```

Comment:
Next time we add one, we should make a loop based on a list (or tuple) of known environment parameters, like the original `make_env` function.

Reply:
:+1: